modules/protocol-tests/resources/META-INF/smithy/manifest

```diff
@@ -7,4 +7,5 @@ HeaderEndpoint.smithy
 Health.smithy
 Pizza.smithy
 RoundTrip.smithy
-Version.smithy
\ No newline at end of file
+Version.smithy
+test-config.json
```

Comment:
How does that work? The file does not exist anymore but we keep a reference to it in the manifest.

Is it possible you moved the file and forgot to include the new one in the commit?

Reply:
just bad commit